migration/src/remap_cross_issue_links.py

```diff
@@ -68,11 +68,11 @@ def remap_issue_link_in_comments(issue_number: int, issue_id_map: dict[str, str]
     check_authentication(github_token)
 
     parser = argparse.ArgumentParser()
-    parser.add_argument('--issues', type=int, required=False, nargs='*', help='GitHub issue number list to be downloaded')
-    args = parser.parse_args()
-    
     mapping_data_dir = Path(__file__).resolve().parent.parent.joinpath(MAPPINGS_DATA_DIRNAME)
     issue_mapping_file = mapping_data_dir.joinpath(ISSUE_MAPPING_FILENAME)
+    parser.add_argument('--issues', type=int, required=False, nargs='*', help=f'GitHub issue number list to be downloaded, else all GitHub issues in {issue_mapping_file.relative_to(Path.cwd())}')
```

Comment:
I copied this help string from another file. Let me adjust it.
```suggestion
    parser.add_argument('--issues', type=int, required=False, nargs='*', help=f'GitHub issue number list to be exported, else all GitHub issues in {issue_mapping_file.relative_to(Path.cwd())}')
```